fluent-plugin-prometheus.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "fluentd", ">= 1.9.1", "< 2"
-  spec.add_dependency "prometheus-client", "< 0.10"
+  spec.add_dependency "prometheus-client", "= 2.1.0"
```

Comment:
`=` is not recommended because users can't use bug fixed version like 2.1.1.
Use `>= 2.1.0` or ` ~> 2.2` like pattern.